code/game/objects/effects/glowshroom.dm

```diff
@@ -153,8 +153,16 @@
 
 /obj/structure/glowshroom/attackby(obj/item/I, mob/user)
 	..()
-	if(I.damtype != STAMINA)
-		endurance -= I.force
+	var/force = I.force
+	if(istype(I, /obj/item/weapon/scythe))
+		var/obj/item/weapon/scythe/S = I
+		if(S.extend)	//so folded telescythes won't get damage boosts / insta-clears (they instead will instead be treated like non-scythes)
+			force = force * 4

```

Comment:
I considered using that, but didn't... probably because it was just a copy-pasted edit of the code on the spacevines.

@RemieRichards you might want to consider putting this up on TG too, since I doubt you guys have done this too since we took your botany.